tests/snmpsim/fujitsueternusos.snmprec

```diff
@@ -0,0 +1 @@
+1.3.6.1.2.1.1.2.0|6|1.3.6.1.4.1.211.1.21.1.150
```

Comment:
Could you add sysDesr as well? I know it's not used in detection but it helps to have more complete os tests.